src/cocotb/types/logic_array.py

```diff
@@ -217,23 +301,75 @@ def count(self, value: Logic) -> int:
         return self._value.count(value)
 
     @property
+    @deprecated("`.binstr` property is deprecated. Use `str(value)` instead.")
     def binstr(self) -> str:
-        return "".join(str(bit) for bit in self)
+        """Converts the value to the :class:`str` literal representation.
+
+        .. deprecated:: 2.0
+        """
+        return str(self)
 
     @property
     def is_resolvable(self) -> bool:
+        """Returns ``True`` if all elements are ``0`` or ``1``."""
```

Comment:
```suggestion
        """Return ``True`` if all elements are ``0`` or ``1``."""
```

Reply:
Actually removing the `return`, missed that this is a property.